OSDOCS-16940: DITA abstracts — OLM admin/user topics (#115359)
openshift/openshift-docs: operators/user/olm-installing-operators-in-namespace.adoc

toc::[]

> [role="_abstract"]
> You can install and subscribe to Operators in your namespace on {product-title} when a cluster administrator has delegated installation permissions. This self-service workflow uses the software catalog in the web console or CLI.

Reviewer: ```suggestion
[role="_abstract"]
If a cluster administrator has delegated Operator installation permissions to your account, you can install and subscribe an Operator to your namespace in a self-service manner.
```
